tao_compiler/mlir/disc/IR/lhlo_disc_ops.td

```diff
@@ -85,6 +85,33 @@ def LHLO_CustomCallOp : LHLODISC_Op<"custom_call", [AttrSizedOperandSegments]> {
   let hasVerifier = 1;
 }
 
+// Models the behavior of the second version BladeDISC custom call.
+// TODO(disc): Merge with the old version once the refactor is done.
```

Comment:
nnnnnnnit: better to have your ID in parentheses after `TODO`.

Reply:
done